docs/app/Examples/elements/Header/Content/HeaderSubheaderExample.js

```diff
@@ -0,0 +1,15 @@
+import React, {Component} from 'react';
+import {Header} from 'stardust';
+
+export default class HeaderSubheaderExample extends Component {
+  render() {
+    return (
+      <Header.H2>
+        Account Settings
+        <div className='sub header'>
+          Manage your account settings and set email preferences
+        </div>
```

Comment:
The `Sub` header class would build up the `ui sub header` class for you.  The theme is responsible for the style (like upper case).